code/index.html

```diff
@@ -3,28 +3,135 @@
+  <input type="text" id="lname" name="lname"  placeholder="Last Name*" required> <br>
+  <label for="Email"></label>
+  <input type="email" id="email" name="email"placeholder="Email*"  required>
+  <label for="phone"></label>
+  <input type="tel" id="phone" name="phone"placeholder="Phone" ><br>
+  <label for="message"></label>
+<textarea name="message" id="message" cols="30" rows="3" placeholder="Leave a Message" ></textarea> <br>
+<label for="checkbox">I Agree to <a href="#">terms & conditions</a> </label>
+ <input type="checkbox" id="checkbox" name="checkbox" required ><br><br>
 
+<input type="submit" id="submit" value="Contact us now" size="10">
+</form>
+</section>
     <section class="accordion">
       <!-- Accordion -->
-      <h1>FAQ</h1>
+      <h2>FAQ's</h2> <br> <br> 
 
-      <div class="question" id="section1">Section 1</div>
+      <button class="question" id="section1">Who are these classes meant for?</button>
       <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas dapibus quis dolor vel imperdiet. Sed tincidunt gravida sem non ornare. Phasellus maximus vestibulum sem nec tempus.</p>
+      </div> <br> <br> 
+
 
-      <div class="question" id="section2">Section 2</div>
+      <button class="question" id="section2">What is the mode of payment for the classes?</button>
       <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas dapibus quis dolor vel imperdiet. Sed tincidunt gravida sem non ornare. Phasellus maximus vestibulum sem nec tempus.</p>
+      </div><br> <br> 
+      <button class="question" id="section2">What are the  pre-requisites for the classes?</button>
+      <div class="answer">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas dapibus quis dolor vel imperdiet. Sed tincidunt gravida sem non ornare. Phasellus maximus vestibulum sem nec tempus.</p> 
+      </div><br> <br> 
+      <button class="question" id="section2">What are the alternative options if the courses I’m interested in are all filled up?</button>
+      <div class="answer">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas dapibus quis dolor vel imperdiet. Sed tincidunt gravida sem non ornare. Phasellus maximus vestibulum sem nec tempus.</p>
+      </div><br>
+    </section>
+    
     </section>
-    <!-- dont forget to link the javascript file here! -->
+    <hr >
+   
+    <footer>
+    
+      
+      <section class= "footer">
+    
+        <p>
+          CONTACT US
+          <i class="fa fa-facebook-square" ></i>
+          <i class="fa fa-envelope" ></i>
+          <i class="fa fa-instagram" ></i><br> <br>
+          
+            <i class="fa fa-copyright"></i>
+            Sherin Thomas week-3 project @ Technigo Front-End Bootcamp 2022 
+        </p>  
+       
+      </section>
+      
+    </footer>
```

Comment:
The code is nice, but u dont need to have a section tag if u already have a footer.
All u dont need to give it a class, u can just write footer i css and it will synk :)